apps/etterlatte-saksbehandling-ui/client/src/components/behandling/virkningstidspunkt/Virkningstidspunkt.tsx

```diff
@@ -76,36 +76,33 @@ const Virkningstidspunkt = (props: {
     if (begrunnelse.trim().length === 0) {
       return setErrorTekst('Begrunnelsen må fylles ut')
     }
-    if (erBosattUtland) {
-      if (!kravdato) {
-        setErrorTekst('Kravdato kreves på bosatt utland saker')
-      }
+    if (erBosattUtland && !kravdato) {
+      return setErrorTekst('Kravdato kreves på bosatt utland saker')
     }
 
-    const harVirkningstidspunktEndretSeg =
-      behandling.virkningstidspunkt?.dato !== formaterDatoTilYearMonth(virkningstidspunkt) ||
-      behandling.virkningstidspunkt?.begrunnelse !== begrunnelse
-    if (harVirkningstidspunktEndretSeg) {
-      return fastsettVirkningstidspunktRequest(
-        { id: behandling.id, dato: virkningstidspunkt, begrunnelse: begrunnelse, kravdato: kravdato },
-        (res) => {
-          dispatch(oppdaterVirkningstidspunkt(res))
-          dispatch(oppdaterBehandlingsstatus(IBehandlingStatus.OPPRETTET))
-          onSuccess?.()
-        },
-        () =>
-          setErrorTekst(
-            'Kunne ikke sette virkningstidspunkt. Last siden på nytt og prøv igjen, meld sak hvis problemet vedvarer'
-          )
-      )
-    } else {
-      onSuccess?.()
```

Comment:
denne ble borte?

Reply:
Ja, fjernet den if'en for å sjekke om det har endret seg - sender til backend uansett når det trykkes på "Lagre"